scope.go

```diff
@@ -72,10 +76,14 @@ type scope struct {
 	tm sync.RWMutex
 	hm sync.RWMutex
 
-	counters   map[string]*counter
-	gauges     map[string]*gauge
-	timers     map[string]*timer
-	histograms map[string]*histogram
+	counters        map[string]*counter
+	countersSlice   []*counter
+	gauges          map[string]*gauge
+	gaugesSlice     []*gauge
+	timers          map[string]*timer
+	timersSlice     []*timer
```

Comment:
Timers aren't used in `CacheReport`, as they report directly.

Reply:
removed and mentioned with a comment